troubleshooting-guide.md

```diff
@@ -1,12 +1,14 @@
 ---
 layout: api-docs
-title:  "Troubleshooting guide"
+title:  "Troubleshooting Guide"
 permalink: /troubleshooting-guide
 in-page-nav: true
 ---
 
 # {{ page.title }}
 
+The troubleshooting guide provides technical assistance to Prescription Drug Plan (PDP) sponsors trying to use the AB2D API. You can also view frequently asked questions on the [Support]({{ '/support' | realtive_url }}) page or contact the AB2D team at [ab2d@cms.hhs.gov](mailto:ab2d@cms.hhs.gov). 
```

Comment:
I don't feel strongly about this, but we can avoid repeating info and shorten this by saying "...use the AB2D API. For further assistance, visit Support."